README.md

```diff
@@ -44,42 +44,53 @@ nightly APIs for operating on slices, which we use while parsing.
 ## Example
 
 ```rust
+use std::sync::Arc;
 use std::time::Duration;
+
 use serde_json::json;
+use url::Url;
 
-use phoenix_channels_client::{Config, Client};
+use phoenix_channels_client::Socket;
 
 #[tokio::main]
 async fn main() {
-    // Prepare configuration for the client
-    let mut config = Config::new("ws://127.0.0.1:9002/socket/websocket").unwrap();
-    config.set("shared_secret", "supersecret");
-
-    // Create a client
-    let mut client = Client::new(config).unwrap();
+    // URL with params for authentication
+    let url = Url::parse_with_params(
+        format!("ws://127.0.0.1:9002/socket/websocket").as_str(),
```

Comment:
This format is unnecessary; you can just use a &'static str here

Reply:
🤔 Why did I use `format!`?  Ah, right, that was from before I found `parse_with_params` and thought I'd have to interpolate the params myself and I never dropped the `format!` after I found it!